demo/demo_spatiotemporal_det.py

```diff
@@ -318,6 +318,11 @@ def main():
     img_norm_cfg['std'] = np.array(img_norm_cfg['std'])
 
     # Build STDET model
+    try:
+        # Fix a issue that different actions may have different bboxes
```

Comment:
# In our spatiotemporal detection demo, different actions should have the same number bboxes.